controller/Publish.php

```diff
@@ -61,6 +63,19 @@ public function publish(){
             $deliveryClass = $this->getClass($classUri);
             /** @var DeliveryFactory $deliveryFactoryResources */
             $deliveryFactoryResources = $this->getServiceManager()->get(DeliveryFactory::SERVICE_ID);
+
+            $qtiTestService = $this->getServiceLocator()->get(TestModelService::SERVICE_ID);
+            $items = $qtiTestService->getItems($test);
+            $lang = \common_session_SessionManager::getSession()->getDataLanguage();
```

Comment:
You can remove the non-namespaced class from the inside of the code name by using a `use` statement with an alias at the top of the file (this also gathers all dependencies together):
```
use \common_session_SessionManager as SessionManager;
...
$lang = SessionManager::getSession()->getDataLanguage();
```